### DISCUSSION TITLE
Launching after the build

### DISCUSSION THREAD

**Discussion by shuntera** [AbdBarho/stable-diffusion-webui-docker#643] [General]
After you have installed everything using --build, from that point on do you just use docker compose up -d to launch from day to day, you don't have to build each time?

And what about performing updates when new versions come out?

**Comment by DevilaN** [comment]
If there is newer version, then you need to use `git pull` command to get latest changes, and then when starting docker will detect that image version tag has changed, so it will rebuild image from the latest version of code.
This is covered by https://github.com/AbdBarho/stable-diffusion-webui-docker/wiki/Usage section.
